source/content/guides/fastly-pantheon/01-introduction.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Fastly on Pantheon
+subtitle: Introduction
+description: Learn more about using Fastly on Pantheon.
+categories: [develop]
+tags: [cms]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/fastly-pantheon
+anchorid: fastly-pantheon
+---
+
+Pantheon partners with [Fastly](https://www.fastly.com/) to bring you a fast and safe content delivery network (CDN).
+
+Using Fastly on the Pantheon platform to manage your site provides many benefits, including:
+
+- Building logic at the edge
+- Content personalization through [Edge Integrations](/guides/edge-integrations/) 
+- Integration with [Splunk](https://www.splunk.com/) and Splunk integration with [New Relic](/new-relic)
```

Comment:
Is there a particular reason we're only referencing Splunk and New Relic?

Reply:
@joebortelpantheon The original issue created for this only referenced Splunk and New Relic. Are there other integrations that our customers use frequently with their Fastly and Pantheon accounts that we can add to this guide?